### DISCUSSION TITLE
LF APAC 开源布道者分享系列之八：开源世界中的横向领导力养成与实践 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#23] [Announcements]
# LF APAC 开源布道者分享系列之八：开源世界中的横向领导力养成与实践 - 开源之道

https://opensourceway.community/posts/lfapac-evangelist-sharing/how-to-lead-when-you-are-not-in-charge/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
Evangelist 其实, 并不是去领导什么,
而是倡导点儿什么, 
现代互联网人, 相比以往每个普通人其实都拥有无限资源,
只是, 一般不知道将精力投放到哪儿去 ;

以及, 这个话题没展开, 只是个书评...

其实, 很危险, 涉及政体基本核心精神: 民主和集中

难以展开, 
其实, 在开源社区中, 很简单的, 能作就能, 不能 BB 再多没人屌你

talk is cheap, show me code.

话糙理不糙, 开源世界中, 很难有外行领导现象,
这也是为什么有能力的愿意在开源世界折腾的原因,
在普通公司, 外行 PM/甲方 实在太恶心了...

所以, 横向领导力, 叕一个原创概念, 其实, 并不存在这种能力, 
只是对普通开发能力, 认真负责到底习惯的又一种解读而已...
